training/general/src/environments/pytorch-1.11-cuda11.3/context/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
+
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.20.2' \
+                'azureml-core=={{latest-pypi-version}}' \
+				'azureml-dataset-runtime=={{latest-pypi-version}}' \
+                'azureml-defaults=={{latest-pypi-version}}' \
+				'azure-ml=={{latest-pypi-version}}' \
+				'azure-ml-component=={{latest-pypi-version}}' \
+                'azureml-mlflow=={{latest-pypi-version}}' \
+                'azureml-telemetry=={{latest-pypi-version}}' \
+		        'azureml-contrib-services=={{latest-pypi-version}}' \
+                'torch-tb-profiler~=0.4.0' \
+				'py-spy==0.3.12' \
+		        'debugpy~=1.6.3'
+
+RUN pip install \
+        azure-ai-ml==1.2.0 \
+        azureml-inference-server-http~=0.7.0 \
+        inference-schema~=1.4.2.1 \
+        MarkupSafe==2.0.1 \
+	    regex \
+	    pybind11
```

Comment:
Please move these to a requirements.txt file, and the update the environment.yaml's `template_files` property. See https://github.com/Azure/azureml-assets/wiki/Environments#template-tags

This also applies to other environments in this PR.

Reply:
added the requirements.txt file for these.